lib/fp.js

```diff
@@ -74,19 +74,20 @@ const replicate = (count, elem) => (Array.from({ length: count }, () => elem));
 
 // Zip arrays using specific function
 //   fn - function, for zipping elements with index i
-//   arrays - array of arrays
-// Returns: array
-// Element with index i of resulting array is result
+//   ...arrays - array, arrays to be zipped
+// Returns: array, zipped
+// Hint: Element with index i of resulting array is result
 // of fn called with arguments from arrays
```

Comment:
I don't think we should use `Hint` keyword, we can just put everything in the place it belongs to (here Result).

Why not 
```
// Returns: array, contains element zipped with provided function. Element at index i of the
//          resulting array is the result of calling fn with arguments at the ith position
//          from input arrays.
```
?
Similar can be applied in other places.

Reply:
Could you please align the `Returns` cases?
i.e
```
Returns: <array>, zipped, element with index i of resulting array is result
         of fn called with arguments from arrays.
```

Also, yeah, doesn't have types in <> for Returns case.